tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/util/AsyncCallerExecutor.java

```diff
@@ -0,0 +1,34 @@
+/*
+ *******************************************************************************
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.eclipse.microprofile.fault.tolerance.tck.util;
+
+import javax.enterprise.context.ApplicationScoped;
+import java.util.concurrent.Executor;
+
+@ApplicationScoped
+public class AsyncCallerExecutor implements Executor {
+
+    private AsyncCaller asyncCaller;
```

Comment:
How is this field initialized? Am I missing something?

Reply:
Nop... you are right. Missing the annotation. 
Which rises the curious question of how the implementation container was successfully passing the test. Will dig tomorrow

Reply:
yup. seems ok now